src/libraries/System.Net.Http/src/System/Net/Http/HttpMessageInvoker.cs

```diff
@@ -41,7 +41,26 @@ public HttpMessageInvoker(HttpMessageHandler handler, bool disposeHandler)
             }
             CheckDisposed();
 
-            return _handler.Send(request, cancellationToken);
+            if (HttpTelemetry.Log.IsEnabled() && !request.WasSentByHttpClient())
+            {
+                try
+                {
+                    return _handler.Send(request, cancellationToken);
+                }
+                catch when (LogRequestAborted(telemetryStarted: true))
+                {
+                    // Unreachable as LogRequestAborted will return false
+                    throw;
+                }
+                finally
+                {
+                    HttpTelemetry.Log.RequestStop();
```

Comment:
Where's the `RequestStart` for this one?

Reply:
Nice catch. I added Telemetry tests for non-async Sends (Invoker.Send & HttpClient.Send).